doc/onboarding-extras.md

```diff
@@ -13,6 +13,7 @@
 | `lib/{crypto,tls,https}` | @nodejs/crypto |
 | `lib/domains` | @misterdjules |
 | `lib/fs`, `src/{fs|file}` | @nodejs/fs |
+| `lib/internal/url` | @nodejs/url |
```

Comment:
Maybe adding a `note` column and mention that `lib/url` doesn't  apply to this would help?